app/helpers/lookup_keys_helper.rb

```diff
@@ -110,38 +111,38 @@ def lookup_key_description(lookup_key, matcher, inherited_value)
   end
 
   def lookup_key_warnings(required, has_value)
-    return { :text => '', :icon => 'info-sign' } if has_value
+    return { :text => '', :icon => 'info-circle' } if has_value
 
     if required
       { :text => _("Required parameter without value.<br/><b>Please override!</b><br/>"),
-        :icon => "warning-sign" }
+        :icon => "exclamation-triangle" }
```

Comment:
Should the required and optional parameters have the same sign? How about:  pficon-error-circle-o?